src/resources/PowerShellWorkerStrings.resx

```diff
@@ -352,4 +352,10 @@
   <data name="DependencySnapshotDoesNotContainAcceptableModuleVersions" xml:space="preserve">
     <value>Dependency snapshot '{0}' does not contain acceptable module versions.</value>
   </data>
+  <data name="GetFunctionsMetadataMultipleResultsError" xml:space="preserve">
```

Comment:
Could you please ping the Team channel to get feedback on the new string resources you added?